src/screens/BeerOfTheDay/styles.scss

```diff
@@ -0,0 +1,33 @@
+.title-beer-day {
+  color: black;
+  font-size: 30px;
+  text-align: center;
+  font-family: fantasy;
+  letter-spacing: 3px;
+  margin-top: 70px;
+}
+
+.button-choose {
+  margin-top: 40px;
+  color: black;
+  width: 100px;
+  height: 100px;
+  border-radius: 50%;
+  outline: 0;
+  font-family: fantasy;
+  border: none;
+  background-color: white;
+  margin-left: 450px;
+  margin-bottom: 30px;
+  font-size: 15px;
+}
+
+.button-choose:hover {
+  background-color: #febc58;
+  box-shadow: -8px -8px 12px 0 rgba(0, 0, 0, 0.3),
+    12px 12px 16px rgba(255, 255, 255, 0.25);
```

Comment:
Oh common. It looks awful - both in the code and on the page.

Reply:
Done!